README.md

```diff
@@ -151,6 +154,32 @@ When added open a PR and once merged to master a github action workflow will sta
 
 Please notice that exporters have their own `build` script but they share the packaging scripts, located under `./scripts`
 
+### <span id="external-files">Adding external config files</span>
+
+When an exporter requires its own config files and does not accept passing the configuration either by CLI arguments or Environment variables, you can add external config files to the exporter definition.
+
+For example, if an exporter requires a `config.toml` configuration file, you can add it by including a `config.toml.tmpl` file together with the other templates described above. This file will also be templated with values from the integration `{exporterName}-config.yml`. To include that file on the build, declare it on the `exporter.yml` definition file under the `exporter_config_files` setting:
+
+```yaml
+# (Optional) Exporter external config files (multiple files can be added separated by commas. i.e. file1,file2)
+exporter_config_files: config.toml
+```
+
+By default, the external config files will be templated and saved on the TEMP folder. If you want to specify a custom location for the external files you can do so by setting `config_path` in your exporter configuration file.
```

Comment:
I'm sorry I didn't realize it in the first review, but I think the name of this default value could be misleading. Maybe we could use something like `exporter_files_path`, WDYT?

Reply:
👍 Agree. is more descriptive